umap/static/umap/js/modules/data/features.js

```diff
@@ -637,35 +633,32 @@ class Feature {
         this._umap.tooltip.open({ content: L._('✅ Copied!') })
       },
     })
-    if (this._umap.editEnabled && !this.isReadOnly()) {
-      items = items.concat(this.getContextMenuEditItems(event))
-    }
     return items
   }
 
   getContextMenuEditItems() {
-    let items = ['-']
-    if (this._umap.editedFeature !== this) {
-      items.push({
-        label: `${translate('Edit this feature')} (⇧+Click)`,
-        action: () => this.edit(),
-      })
-    }
-    items = items.concat(
+    return [
       {
-        label: this._umap.help.displayLabel('EDIT_FEATURE_LAYER'),
+        action: () => this.toggleEditing(),
+        title: translate('Toggle edit mode (⇧+Click)'),
```

Comment:
I don't think that's the right title for that icon. We are already in edit mode and the keyboard shortcut is not really useful at that point:
<img width="218" height="191" alt="Capture d’écran, le 2025-07-22 à 09 56 54" src="https://github.com/user-attachments/assets/4496c7d7-3a38-4c8b-a691-e9b1b4319b3f" />

I would just let “Edit” (or the more complex “Edit this …”)